mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -233,6 +236,8 @@ pub enum Error {
     },
     #[snafu(display("Invalid ShareProtocol value {}", sp_value))]
     InvalidShareProtocol { sp_value: i32 },
+    #[snafu(display("Invalid NvmeAnaState value {}", ana_value))]
+    InvalidNvmeAnaState { ana_value: i32 },
```

Comment:
this i32 could be the NvmeAnaState

Reply:
Not really as the user only has an `i32` plus if it's invalid we'd want to know what the value was.